bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -0,0 +1,2172 @@
+integration.define({
+	synchronizations: [
```

Comment:
Since this is a maintenance bug fix for customer issue, we didn't get approval from PM to work on additional features for this integration. So we haven't implemented incremental sync here.